docusaurus/docs/operate/morse_migration/11_testing.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migration Testing
+sidebar_position: 11
+---
+
+## Table Of Contents
+
+- [Background](#background)
+- [1. Retrieve a Morse TestNet Snapshot](#1-retrieve-a-morse-testnet-snapshot)
+- [2. Export Morse TestNet Export State](#2-export-morse-testnet-export-state)
+- [3. Transform & Merge Morse Exports to a Canonical Account State Import Message](#3-transform--merge-morse-exports-to-a-canonical-account-state-import-message)
+
+## Background
+
+For testing the migration process end-to-end, _on a live and public network and with real stakeholders_, the Morse state
+is initially imported into Shannon TestNet(s).
```

Comment:
```suggestion
must be initially imported into Shannon TestNet(s).
```
Should it be a "must be" instead?

Reply:
:thinking: Not sure. In my view, we're describing our operational decision, not justifying its philosophy. I.e., I think it's fine as is. :sweat_smile: